Week3/akif-week3.js

```diff
@@ -0,0 +1,158 @@
+//   STRINGS! --------------
+let myString = 'hello,this,is,a,difficult,to,read,sentence';
+console.log(myString);
+console.log(myString.length);
+myString = myString.replace(/[,]/g, ' ');
+console.log(myString);
+
+//   ARRAYS! -------------------------------------------------
+let favoriteAnimals = ['blowfish', 'capricorn', 'giraffe'];
+favoriteAnimals.push('turtle');
+console.log(favoriteAnimals);
+favoriteAnimals.splice(1, 0, 'meerkat');
+console.log("i added a new value 'meerkat' to the array");
+console.log(favoriteAnimals);
+console.log('The array has a length of: ' + favoriteAnimals.length);
+favoriteAnimals.splice(3, 1);
+console.log(favoriteAnimals);
+console.log('The item you are looking for is at index: ' + favoriteAnimals.indexOf('meerkat'));
+favoriteAnimals.splice(favoriteAnimals.indexOf('meerkat'), 1);
+
+//  MORE JAVASCRIPT ! ------------------------------------------------
+// -----1111111111111111111-----------
+function getSum(a, b, c) {
+  return a + b + c;
+}
+
+// -----222222222222222222-------------
+function colorCar(color) {
+  return 'a ' + color + ' car';
+}
+
+//-----3333333333333333333--------------
+const myBooks = {
+  Dostoyevsky: 'Crime and Punishment',
+  'Franz Kafka': 'The Trial',
+  'Anne Frank': 'Diarys',
+};
+
+function getBook(library) {
+  for (let book in library) {
+    console.log(book + ' - ' + library[book]);
```

Comment:
Hi Akif this function works fine. However, if you remember from our class, I mentioned that you can try to use `Object.keys` instead of a `for-in` loop. You can perhaps try to google the difference between the two approaches and see if it makes sense. Otherwise, I think this is going to be covered in JS3. 🙂

Reply:
Thanks for remarking subtle issues like that. Also for other comments. I think your comments is very useful 👍